diffpy/snmf/factorizers.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+import scipy.optimize
+
+
+def lsqnonneg(coefficient, target):
+    """
+    Solve ``argmin_x || Ax - b ||_2`` for ``x>=0``.
+
+    Parameters
+    ----------
+    coefficient: 2-dimensional ndarray or list
```

Comment:
`ndarray` means "n-dimensional array" so `2d array like` is enough.

To me this docstring is still confusing "The coefficient matrix of the least squares problem" doesn't convey to me what should be in each of the two dimensions of the array (and in which order)